nginx/src/css/sidebar.css

```diff
@@ -1,37 +1,127 @@
-#sidebar-wrapper {
-	position: fixed;
-	top: 0;
-	left: 0;
+#sidebar {
+	box-sizing: border-box;
 	height: 100vh;
 	width: 250px;
-	border-radius: 0 5px 5px 0;
-	transition: all 0.3s ease;
-	z-index: 1000;
-	text-align: center;
+	padding: 5px 1em;
+	background-color: var(--base-clr);
+	border-right: 1px solid var(--line-clr);
+
+	position: sticky;
+	top: 0;
+	align-self: start;
+	transition: 300ms ease-in-out;
+	overflow: hidden;
+	text-wrap: nowrap;
+}
+
+#sidebar.close {
+	padding: 5px;
+	width: 60px;
 }
 
-#sidebar-wrapper.toggled {
-	margin-left: -250px;
+#sidebar ul {
+	margin-left: -2.6rem;
+	list-style: none;
 }
 
-#sidebar-wrapper .sidebar-heading {
-	padding: 1rem;
-	font-size: 1.5rem;
+#sidebar>ul>li:first-child {
+	display: flex;
+	justify-content: flex-end;
+	margin-bottom: 16px;
+
+	.logo {
+		font-weight: 600;
+	}
 }
 
-#sidebar-wrapper .list-group-item {
-	padding: 1rem 1.5rem;
+/* #sidebar ul li.active a {
+	color: var(--accent-clr);
+
+	svg {
+		fill: var(--accent-clr);
+	}
+} */
+
+#sidebar a,
+#sidebar .dropdown-btn,
+#sidebar .logo {
+	border-radius: .5em;
+	padding: .85rem;
+	text-decoration: none;
+	color: var(--text-clr);
+	display: flex;
+	align-items: center;
+	gap: 1em;
 }
 
-#sidebar-toggle {
-	position: fixed;
-	top: 1rem;
-	left: 1rem;
-	font-size: 20px;
-	z-index: 1050;
-	transition: all 0.3s ease;
+.dropdown-btn {
+	width: 100%;
+	text-align: left;
+	background: none;
+	border: none;
+	font: inherit;
+	cursor: pointer;
 }
 
-#sidebar-toggle.toggled {
-	margin-left: 250px;
+#sidebar svg {
+	flex-shrink: 0;
+	fill: var(--text-clr);
+}
+
+#sidebar a span,
+#sidebar .dropdown-btn span {
+	flex-grow: 1;
+}
+
+#sidebar a:hover,
+#sidebar .dropdown-btn:hover {
+	background-color: var(--hover-clr);
+}
+
+#sidebar .sub-menu {
+	display: grid;
+	grid-template-rows: 0fr;
+	transition: 300ms ease-in-out;
+
+	>div {
```

Comment:
not valid css

Reply:
adressed in : [85e156a](https://github.com/TheSecretOrganization/ft_transcendence/pull/201/commits/85e156af24302898ff928ba7a46c4a1a9e8c5210)